src/core_plugins/kibana/server/routes/api/management/saved_objects/relationships.js

```diff
@@ -0,0 +1,46 @@
+import Boom from 'boom';
+import Joi from 'joi';
+import _ from 'lodash';
+import { findRelationships } from '../../../../lib/management/saved_objects/relationships';
+
+export function registerRelationships(server) {
+  server.route({
+    path: '/api/kibana/management/saved_objects/relationships/{type}/{id}',
+    method: ['GET'],
+    config: {
+      validate: {
+        params: Joi.object().keys({
+          type: Joi.string(),
+          id: Joi.string(),
+        }),
+        query: Joi.object().keys({
+          size: Joi.number(),
+        })
+      },
+    },
+
+    handler: async (req, reply) => {
+      const { callWithRequest } = server.plugins.elasticsearch.getCluster('data');
+      const boundCallWithRequest = _.partial(callWithRequest, req);
+
+      const type = req.params.type;
+      const id = req.params.id;
+      const size = req.query.size || 10;
+
+      try {
+        const response = await findRelationships(
+          type,
+          id,
+          size,
+          boundCallWithRequest,
+          req.getSavedObjectsClient(),
+        );
+
+        reply(response);
+      }
+      catch (err) {
+        reply(Boom.boomify(err, { statusCode: 500 }));
```

Comment:
(Not sure if this comment is in the right file)
If I have a search, and delete the index pattern associated with the search, the Relationship flyout view shows server error. Is that expected?

<img width="1124" alt="screen shot 2018-03-30 at 12 36 10 pm" src="https://user-images.githubusercontent.com/1965714/38151201-2cd235dc-3417-11e8-87ee-0892e3082f29.png">

Reply:
Fixed!